include/beast/core/impl/basic_streambuf.ipp

```diff
@@ -582,7 +582,7 @@ basic_streambuf<Allocator>::prepare(size_type n) ->
         list_.push_back(e);
         if(out_ == list_.end())
             out_ = list_.iterator_to(e);
-        if(n > e.size())
+        if(n >= e.size())
```

Comment:
I don't see where this fixes anything. It's correct as written, but as far as I can tell the old code was also correct. Why the change?